DER/DerivativesContracts/RightsAndWarrants.rdf

```diff
@@ -132,6 +204,78 @@
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-raw;SubscriptionRight"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;hasIdentity"/>
+						<owl:someValuesFrom rdf:resource="&fibo-sec-eq-eq;ConvertibleCommonShare"/>
+					</owl:Restriction>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">common convertible share subscription right</rdfs:label>
+		<skos:definition xml:lang="en">subscription right entitling existing common convertible shareholders to subscribe to new securities at a price normally lower than the prevailing market price</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom xml:lang="en">ISO 10962, Securities and related financial instruments - Classification of financial instruments (CFI) code, Fourth Edition, October 2019</fibo-fnd-utl-av:adaptedFrom>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-drc-raw;CommonSharePurchaseRight">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-raw;PurchaseRight"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;hasIdentity"/>
+						<owl:someValuesFrom rdf:resource="&fibo-sec-eq-eq;CommonShare"/>
+					</owl:Restriction>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">common share purchase right</rdfs:label>
+		<skos:definition xml:lang="en">purchase right that gives a prospective acquiree&apos;s common shareholders the right to buy common shares of the firm or common shares of anyone who acquires the firm at a deep discount to their fair market value</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom xml:lang="en">ISO 10962, Securities and related financial instruments - Classification of financial instruments (CFI) code, Fourth Edition, October 2019</fibo-fnd-utl-av:adaptedFrom>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-drc-raw;CommonShareSubscriptionRight">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-raw;SubscriptionRight"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;hasIdentity"/>
+						<owl:someValuesFrom rdf:resource="&fibo-sec-eq-eq;CommonShare"/>
+					</owl:Restriction>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">common share subscription right</rdfs:label>
+		<skos:definition xml:lang="en">subscription right entitling existing common shareholders to subscribe to new securities at a price normally lower than the prevailing market price</skos:definition>
```

Comment:
Surely the subscription right gives the holder the right to BUY the shares. Saying they have the right to "subscribe" to the shares is unhelpful and meaningless since we don't define "subscribe" AFAIK, and don't need to.
Should explain that this is generally to avoid dilution due to secondary share offerings. And that the number they can purchase is directly related to their existing holding. And that the right might not be transferable.
See https://www.investopedia.com/terms/s/subscription_right.asp